File: fizzbuzz.php
<?php
  for ($x=1;$x<=100;$x++){
    $o="";
    if ($x % 3 == 0){
      $o.="Fizz";
    }
    if ($x % 5 == 0){
      $o.="Buzz";
    }

    if ($o==""){
      $o=$x;
    }

    echo "$o\r\n";
  }

File: fizzbuzz.py
# Written/tested with Python 3.7.2 on Windows
# The beauty of Python, is that almost everything can be done on a single line ;-)
print '\n'.join("Fizz"*(i%3==0)+"Buzz"*(i%5==0) or str(i) for i in range(1,101)) 
